src/api/1/controller-server/get-config/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: full.html.hbs
+algolia: true
+title: getConfig
+---
+
+# getConfig
+
+{{{since "1.0.0"}}}
+
+Return the current Kuzzle configuration.
+
+This route should only be accessible to administrators, as it might return sensitive information about the backend.
```

Comment:
Shouldn't we highlight this somehow?

Reply:
I don't think this is needed anymore. Descriptions are now simple and short, making that kind of warning stand out sufficiently.